core/modules/savers/git-service.js

```diff
@@ -61,24 +58,27 @@ GitHubSaver.prototype.save = function(text,method,callback) {
 			if(err && xhr.status !== 404) {
 				return callback(err);					
 			}
+			var requestType = service.requestTypeIfFileExists;
 			if(xhr.status !== 404) {
 				getResponseData = JSON.parse(getResponseDataJson);
 				$tw.utils.each(getResponseData,function(details) {
 					if(details.name === filename) {
 						sha = details.sha;
 					}
 				});				
+			} else {
+				requestType = service.requestTypeIfFileNotExists;
 			}
-			var data = {
-					message: "Saved by TiddlyWiki",
-					content: base64utf8.base64.encode.call(base64utf8,text),
-					branch: branch,
-					sha: sha
-				};
+			var data = service.formatDataForCreatingCommit(
+				"Saved by TiddlyWiki",
```

Comment:
While we're working on this, we might want to make this string be translatable.

Reply:
Ok, but then we need to change it in the other savers as well.

Reply:
Should I convert to lingo in each saver in this PR or should I open another request?

Reply:
Sorry, I misunderstood what to translate, corrected it.